WalletWasabi.Gui/Controls/WalletExplorer/ReceiveTabViewModel.cs

```diff
@@ -42,7 +42,7 @@ public ReceiveTabViewModel(Wallet wallet)
 				LabelSuggestion.Label = label;
 				if (label.IsEmpty)
 				{
-					NotificationHelpers.Warning("Observers are required.");
+					NotificationHelpers.Warning("Observers are required.", sender: Wallet);
```

Comment:
Probably this is redundant in this case because the context is clear.  It doesn't hurt anyway.